server/src/main/java/org/opentosca/toscana/plugins/kubernetes/docker/dockerfile/builder/commands/DockerfileEntry.java

```diff
@@ -0,0 +1,12 @@
+package org.opentosca.toscana.plugins.kubernetes.docker.dockerfile.builder.commands;
+
+import java.io.PrintWriter;
+
+public abstract class DockerfileEntry {
+    public abstract void appendToDockerfile(
```

Comment:
I do not like that `previous`, `next` and `compact` are used in only one implementation. But I cannot come up with a better idea.

Reply:
Well thats why i added them here